### ISSUE TITLE
feature request: eraticate twitch

### ISSUE THREAD

**Issue by nikolayganovski1** [jordwest/news-feed-eradicator#144]
can you add option to eradicate twitch.tv site?
thanks.

**Comment by brandon-mork** [comment]
Added a PR for this, awaiting @jordwest 's approval!

**Comment by jNsatiable** [comment]
Yes, please. This is the only thing missing in my newsfeed eradicating spree.

**Comment by alexeiidonetskiy** [comment]
> Yes, please. This is the only thing missing in my newsfeed eradicating spree.

+1
